backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -504,9 +504,9 @@ internal class FunctionGenerationContext(val function: LLVMValueRef,
     fun allocArrayOnStack(arrayTypeName: String, count: LLVMValueRef): LLVMValueRef {
         val arraySlot = alloca(localArrayType(arrayTypeName, extractConstUnsignedInt(count).toInt()))
         // Set array size in ArrayHeader.
-        val arrayHeaderSlot = structGep(arraySlot, 0)
+        val arrayHeaderSlot = structGep(arraySlot, 0, "arrayHeaderField")
```

Comment:
I meant exactly the name of a field from the declaration in runtime. With that we can grep it if want to find all usages.

Reply:
There is no declaration in runtime for this case. It's generated class for local array. In other cases I'll fix.